index.src.html

```diff
@@ -17,96 +17,99 @@
+
+Broadly, <a>authenticators</a> are of two types:
 
 1. <dfn>Embedded authenticators</dfn> have their operation managed by the same computing device (e.g., smart phone, tablet,
     desktop PC) as the user agent is running on. For instance, such an authenticator might consist of a Trusted Platform Module
-    (TPM) or Secure Element (SE) integrated into the computing device, along with appropriate platform software to mediate
-    access to this device's functionality.
+    (TPM) or Secure Element (SE) integrated into the computing device in conjunction with some biometric sensor (e.g., fingerprint), along with appropriate platform software to mediate
+    access to these components' functionality.
 
-2. <dfn>External authenticators</dfn> operate autonomously from the device running the user agent, and accessed over a transport
-    such as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC).
+2. <dfn>External authenticators</dfn> operate autonomously from the computing device running the user agent, and are accessed over a transport
+    such as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC). 
 
 Note that an external authenticator may itself contain an embedded authenticator. For example, consider a smart phone that
 contains a scoped credential. The credential may be accessed by a web browser running on the phone itself. In this case the
 module containing the credential is functioning as an embedded authenticator. However, the credential may also be accessed over
 BLE by a user agent on a nearby laptop. In this latter case, the phone is functioning as an external authenticator. These modes
-may even be used in a single end-to-end user scenario. One such scenario is described in the remainder of this section.
+may even be used in a single end-to-end user scenario. One such scenario is described among the use cases below (in [[#authentication-external]]).
+
 
+## Use Cases ## {#use-cases}
 
-## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
+The below use case scenarios illustrate both <a>embedded authenticator</a> and <a>external authenticator</a> modes, as well as outline further scenarios. 
```

Comment:
sort of -- some readers may find it useful that the Use Cases have a ToC entry.  Also, I'll add to that intro text that there are further use case + code examples in a later spec section (in order to address one of AdamP's comments), ok?